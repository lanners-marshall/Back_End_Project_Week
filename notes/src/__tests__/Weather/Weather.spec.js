import React from 'react';
import ReactDOM from 'react-dom';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Weather from '../../components/Weather/Weather';
import renderer from 'react-test-renderer';

describe('<Weather />', () => {
  it('should match snapshot', () => {
	  const wrapper = shallow(
	      <Weather />
  	)
  	const tree = renderer.create(wrapper).toJSON()
  	expect(tree).toMatchSnapshot()
	})
});