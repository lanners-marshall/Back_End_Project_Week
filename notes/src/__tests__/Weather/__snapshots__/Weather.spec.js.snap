// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Weather /> should match snapshot 1`] = `
<div>
  <div
    className="sc-bxivhb iOYudn"
  >
    
  </div>
  <div
    className="sc-bdVaJa chOuRa"
  />
  <div
    className="sc-htpNat crDpXs"
  >
    <form>
      <input
        name="inputCity"
        onChange={[Function]}
        placeholder="City"
      />
      <input
        name="country"
        onChange={[Function]}
        placeholder="Country Code, us, uk ect"
        value=""
      />
    </form>
    <button
      onClick={[Function]}
    >
      check your local weather
    </button>
  </div>
</div>
`;
